flytekit/clients/friendly.py

```diff
@@ -1010,14 +1010,15 @@ def get_upload_signed_url(
                 expires_in_pb = Duration()
                 expires_in_pb.FromTimedelta(expires_in)
             return super(SynchronousFlyteClient, self).create_upload_location(
-                _data_proxy_pb2.CreateUploadLocationRequest(
+                flyteidl.service.CreateUploadLocationRequest(
                     project=project,
                     domain=domain,
                     content_md5=content_md5,
                     filename=filename,
                     expires_in=expires_in_pb,
-                    filename_root=filename_root,
+                    filename_root=filename_root or "",
                     add_content_md5_metadata=add_content_md5_metadata,
+                    org="",
```

Comment:
initialize `org`  as empty string, can be assigned later by union serverless sdk